react/features/toolbox/middleware.js

```diff
@@ -28,6 +32,13 @@ MiddlewareRegistry.register(store => next => action => {
         break;
     }
 
+    case SET_CONFIG:
+    case OVERWRITE_CONFIG:
+        if (config && typeof config.toolbarAlwaysVisible === 'boolean') {
+            dispatch(setToolboxAlwaysVisible(config.toolbarAlwaysVisible));
```

Comment:
Let's not do this. `setToolboxAlwaysVisible` is currently not used, so let's stop having the duplicated state (config.toolboxAlwaysVisible vs the toolbox reducer state) and it will react to changes automagically.

Reply:
While you're at it you'll notice `setToolboxAlwaysVisible` is declared in actions.native.js, which is imported by actions.web.js. That is BAD! (not your doing though) So let's take the  chance and split the common parts to some actions.any.js.